HW19/js/script.js

```diff
@@ -0,0 +1,65 @@
+ var counter = 0, sum = 0;
+
+function gcollege() {
+    let data = [];
+    data[0] = document.getElementById("sname");
+    data[1] = document.getElementById("sgrade");
+    Avg_s= document.getElementById("avg_students");
+    Count_s = document.getElementById("num_students");
```

Comment:
I don't see any declaration of any variable of `Avg_s` and `Count_s `

Reply:
Hey Elias, 
I did the declaration to the missing variables.
The code works ,  that's why I didn't notice..